### PR TITLE
Attribute creator in polls

### PR DESCRIPTION
**Pull request by scheibo**
**Context:** https://www.smogon.com/forums/threads/let-users-see-who-made-a-poll-or-hangman.3652444/

image

### CODE REVIEW

File: server/chat-plugins/poll.js [hunk 1]
```diff
@@ -218,6 +222,7 @@ exports.Poll = Poll;
 /** @type {ChatCommands} */
 const commands = {
 	poll: {
+		anoncreate: 'new',
```
Comment (TheJetOU) [inline server/chat-plugins/poll.js]: That means you can't create anonymous HTML polls.

You'll probably need to make another alias (`anonhtmlcreate`) and handle it accordingly.

**Comment by bumbadadabum** [comment]
But why? What's the point?

**Comment by Zarel** [comment]
I think a "Foo created a poll" public message would be nice at the start of a poll, but as-is this feels like a waste of screen real-estate. Polls are already too spammy as-is.

**Comment by Slayer95** [comment]
There are rooms give attribution to the _user who submits the question_ for staff to publish. That already uses up some space. Implementing support for that _might_ be more useful? Still, staff already handle that with the currently available tools...

So, I don't think this is needed. +1 for a single notification when the poll starts.

File: server/chat-plugins/poll.js (hunk 1 above)
Comment (scheibo) [reply server/chat-plugins/poll.js]: Yeah, I saw that, just didnt think it mattered.

**Comment by scheibo** [comment]
From the suggestion it seemed the main support for the use case was not having a bunch of users asking "who created the poll?". I'm not sure a public message accomplishes that for users the join the room after the fact (because polls can persist much longer than default scrollback, no?). Anyway, whatever, idc. Someone else can try this.

**Pull request closed without merging**